### ISSUE TITLE
Test failure when run with environment variable ETS_TOOLKIT=qt

### ISSUE THREAD

**Issue by kitchoi** [enthought/pyface#832]
From this comment, https://github.com/enthought/pyface/issues/560#issuecomment-657463578, setting ETS_TOOLKIT to "qt" should still work (however the same cannot be said for, e.g. enable).

However when I run the test suite with `export ETS_TOOLKIT="qt"`, I get some test failures.

The commands I ran:
```
$ python3.9 -m venv ../py39-venv/pyface-env
$ source ../py39-venv/pyface-env/bin/activate
$ pip install .[pyside2,test]
$ cd
$ export ETS_TOOLKIT=qt
$ python -m unittest discover pyface
```
Test failure:
```
======================================================================
FAIL: test_find_current_toolkit_no_etsconfig (pyface.tests.test_base_toolkit.TestToolkit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kchoi/Work/ETS/pyface/pyface/tests/test_base_toolkit.py", line 34, in test_find_current_toolkit_no_etsconfig
    self.assertEqual(toolkit.toolkit, old_etsconfig_toolkit)
AssertionError: 'qt4' != 'qt'
- qt4
?   -
+ qt
```

(There is another orthogonal test error to do with importing wx, see #828)

The same error can be reproduced with `etstool.py` with the environment variable `ETS_TOOLKIT` changed to `"qt"` (it is set by the test command).

**Comment by kitchoi** [comment]
Note that if I run the test module on its own, the error message is different:
```
$ python -m unittest pyface.tests.test_base_toolkit
F....
======================================================================
FAIL: test_find_current_toolkit_no_etsconfig (pyface.tests.test_base_toolkit.TestToolkit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kchoi/Work/ETS/py39-venv/pyface-env/lib/python3.9/site-packages/pyface/tests/test_base_toolkit.py", line 34, in test_find_current_toolkit_no_etsconfig
    self.assertEqual(toolkit.toolkit, old_etsconfig_toolkit)
AssertionError: 'qt4' != None

----------------------------------------------------------------------
Ran 5 tests in 0.175s
```

**Comment by aaronayres35** [comment]
I followed the procedure described in the issue description, only instead using pyface 7.1.0, and still saw the issue.  With that, I am going to proceed with the 7.2 release and leave this issue to be addressed for a future release.

**Comment by corranwebster** [comment]
This is resolved by #1227
